templates/_partials/user/account_dropdown.html.twig

```diff
@@ -8,6 +8,19 @@
         {% trans %}navbar.myAccount{% endtrans %}
       </a>
     </li>
+
+    {% set queryParams = app.request.query.all %}
+    {% if context_channel.channel.code == 'pro' %}
+      {% set queryParams = queryParams|merge({change_channel: 'web'}) %}
+      {% set accountType = 'navbar.lambdaCart' %}
+    {% else %}
+      {% set queryParams = queryParams|merge({change_channel: 'pro'}) %}
+      {% set accountType = 'navbar.proCart' %}
+    {% endif %}
+    {% set currentPath = path(app.request.attributes.get('_route'),
+      app.request.attributes.get('_route_params')|merge(queryParams)) %}
+    <li><a href="{{ currentPath }}">{{ accountType|trans }}</a></li>
```

Comment:
Only "authorized" users should be able to switch channels, i.e users belonging to a group associated with an Organization